docs/Model_Assumptions.rst

```diff
@@ -115,32 +115,34 @@ Constraints
 
 Constraints are controlled with the file `constraints.csv`.
 
-Minimal renewable share constraint
+Minimal renewable factor constraint
 ##################################
 
-The minimal renewable share constraint requires the capacity and dispatch optimization of the MVS to reach at least the minimal renewable share defined within the constraint. The renewable share of the optimized energy system may also be higher then the minimal renewable share.
+The minimal renewable factor constraint requires the capacity and dispatch optimization of the MVS to reach at least the minimal renewable factor defined within the constraint. The renewable share of the optimized energy system may also be higher then the minimal renewable factor.
```

Comment:
```suggestion
The minimal renewable factor constraint requires the capacity and dispatch optimization of the MVS to reach at least the minimal renewable factor defined within the constraint. The renewable share of the optimized energy system may also be higher than the minimal renewable factor.
```

Reply:
I am not sure what "defined within the constraint" refers to? Is it linked to the constrains.csv file?